ntp-proto/src/config.rs

```diff
@@ -266,9 +263,6 @@ pub struct SynchronizationConfig {
     /// synchronizing the clock
     #[serde(default = "default_local_stratum")]
     pub local_stratum: u8,
-
-    #[serde(default)]
-    pub algorithm: AlgorithmConfig,
```

Comment:
I believe this change requires an update to the ntp.toml manpage.

Reply:
This isn't needed because of flattening in the actual read configuration struct.